lib/fluent/plugin/in_groonga.rb

```diff
@@ -132,7 +131,6 @@ def configure(conf)
 
       def start
         listen_socket = TCPServer.new(@bind, @port)
```

Comment:
For **minimal** multiple workers support, I guess that we can use `server_create_tcp_socket(shared, bind, port)` helper method in server plugin helper.